### PR TITLE
Update henry.py

### PR DESCRIPTION
**Pull request by wanda-carlson**

### CODE REVIEW

**Comment by wanda-carlson** [comment]
Test

**Comment by wanda-carlson** [comment]
Can someone review this?

**Comment by wanda-carlson** [comment]
@henry-richard Can you review this?

**Comment by henry-richard** [comment]
Indeed

File: src/henry.py [hunk 1]
```diff
@@ -1 +1,3 @@
 print("Henry testing v0.8.0")
+print("Henry testing")
```
Comment (henry-richard) [inline src/henry.py:2]:
```suggestion
print("Wanda testing")
```

**Review by henry-richard — CHANGES_REQUESTED** [review]

**Review by henry-richard — APPROVED** [review]

**Pull request merged**